Makefile

```diff
@@ -1,58 +1,323 @@
-KFVER=131
+#
+# Semantic versioning
+#
+# KFVER_MAJOR denotes the ABI version.
+#
+# - It must be bumped only if API public members are removed or
+#   changed in the incompatible
+#
+# KFVER_MINOR denotes the minor version within a compatible ABI.
+#
+# - It should be bumped if new API public members are added
+#   (but not removed!) so programs linked against the same library
+#   version continue operating properly
+#
+# KFVER_PATCH denotes bugfix count since the last minor update.
+#
+# - It should be bumped whenever a bug fix is pushed.
+#
 
-DATATYPE ?= float
+export KFVER_MAJOR = 131
+export KFVER_MINOR = 1
+export KFVER_PATCH = 0
+
+#
+# Data type (float / int16_t / int32_t / simd)
+#
+
+export KISSFFT_DATATYPE ?= float
+
+#
+# Default options
+#
+
+export KISSFFT_OPENMP ?= 0
+export KISSFFT_STATIC ?= 0
+export KISSFFT_TOOLS ?= 1
+export KISSFFT_USE_ALLOCA ?= 0
+
+#
+# Installation directories
+#
 
 PREFIX ?= /usr/local
-INCDIR ?= $(PREFIX)/include
-LIBDIR ?= $(PREFIX)/lib
+export ABS_PREFIX = $(abspath $(PREFIX))
+
+BINDIR ?= $(ABS_PREFIX)/bin
+export ABS_BINDIR = $(abspath $(BINDIR))
+
+INCDIR ?= $(ABS_PREFIX)/include
```

Comment:
Maybe this could be renamed to `includedir` to better match the GNU variable names.

Reply:
Then also `bindir`, `includedir`, `libdir` and `pkgincludedir`. And `ABS_` or `abs_` ? :)

Reply:
I think the casing is fine for makefiles (with pure make, the capital casing is slightly more common, in my experience). I mostly meant inc → include.

Reply:
OK, pushed the fix replacing `INCDIR` witn `INCLUDEDIR` :)